src/cclib/io/ccio.py

```diff
@@ -40,7 +40,7 @@
 from ..parser.psiparser import Psi
 from ..parser.qchemparser import QChem
 
-from . import cjsonreader
+from .cjsonreader import CJSON
```

Comment:
I the previous import was better. Use `cjsonread.CJSON below` for consistency. It would be nice to merge the reader and writer into one class in the future.